sql/src/test/java/io/crate/testing/SQLExecutor.java

```diff
@@ -128,6 +133,7 @@ public SQLExecutor build() {
             if (!blobTables.isEmpty()) {
                 schemas.put(BlobSchemaInfo.NAME, new BlobSchemaInfo(clusterService, new TestingBlobTableInfoFactory(blobTables)));
             }
+            File tempDir = createTempDir();
```

Comment:
This tempDir is never deleted this way.

Maybe make the SQLExecutor closeable or make the user responsible for providing an environment/tempfolder ?